airflow/www/templates/airflow/dag.html

```diff
@@ -216,6 +216,9 @@ <h4 class="pull-right js-dataset-triggered" style="user-select: none;-moz-user-s
           <li id="code-nav"><a href="{{ url_for('Airflow.code', dag_id=dag.dag_id, root=root) }}">
             <span class="material-icons" aria-hidden="true">code</span>
             Code</a></li>
+          <li id="code-analysis"><a href="{{ url_for('Airflow.code_analysis', dag_id=dag.dag_id, root=root) }}">
+            <span class="material-icons" aria-hidden="true">live_help</span>
+            Airflow Doctor DAG Code Analysis</a></li>
```

Comment:
shall we change "Airflow Doctor DAG Code Analysis" to "Code Analysis"

Reply:
Sure. I think I was thinking to change the same of Airflow Doctor to Airflow Assistant as well.